src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const SPOUSE = sex === 'f' ? 'husband' : 'wife';
+  const NOT_MARRIED_MESSAGE = 'I am not married';
```

Comment:
- The "SPOUSE" variable is not constant as it is set dynamically, so they should be in camelCase
- NOT_MARRIED_MESSAGE constant you can move outside the component